renderdoc/driver/gl/gl_shader_refl.cpp

```diff
@@ -2278,6 +2278,8 @@ void MakeShaderReflection(GLenum shadType, GLuint sepProg, ShaderReflection &ref
           sig.systemValue = ShaderBuiltin::GSInstanceIndex;
         if(IS_BUILTIN("gl_Layer"))
           sig.systemValue = ShaderBuiltin::RTIndex;
+        if(IS_BUILTIN("gl_ViewIndex"))
+          sig.systemValue = ShaderBuiltin::MultiViewIndex;
```

Comment:
FWIW this extension as a core feature never existed on GL. There is a vendor extension OVR_multiview extension that some other vendors implemented, its builtin is `gl_ViewID_OVR` so if you're adding it here then it should be checking for that name.